.gitpod.yml

```diff
@@ -1,6 +1,13 @@
 image: gitpod/workspace-full
 
 tasks:
+  - env:
+      # https://github.com/gitpod-io/gitpod/issues/479
+      PIP_USER: no
+  - init:
+      brew install pre-commit
+      pre-commit install
+      pre-commit run
```

Comment:
This is failing for me:

```
$ pre-commit run
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/jumanjihouse/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/igorshubovych/markdownlint-cli.
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: CalledProcessError: command: ('/home/gitpod/.cache/pre-commit/repopsdnf9zr/py_env-python3.9/bin/python', '-mpip', 'install', '.')
return code: 1
expected return code: 0
stdout: (none)
stderr:
    ERROR: Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
```

Reply:
The key is PIP_USER: no

```
# https://github.com/gitpod-io/gitpod/issues/479
PIP_USER: no
```